import-export-cli/cmd/aws/init.go

```diff
@@ -0,0 +1,512 @@
+var getRestAPIsCmdOutput string 
+var err error 
+var awsInitCmdForced bool
+var initCmdOutputDir string
+
+//common aws cmd flags 
+var apiGateway string = "apigateway"
+
+//aws cmd Output type
+var outputFlag string = "--output"
+var outputType string = "json"
+
+//cmd_1 
+var awsCLIVersionFlag string = "--version"
+
+//getRestAPIsCmd
+var getAPI string = "get-rest-apis"
+
+//getExportCmd
+var getExport string = "get-export"
+var apiIdFlag string = "--rest-api-id"
+var stageNameFlag string = "--stage-name"
+var exportTypeFlag string = "--export-type"
+var exportType string = "oas30"	//default export type is openapi3. Use "swagger" to request for a swagger 2.
+var debugFlag string			//aws cli debug flag for apictl verbose mode 
+
+const awsInitCmdLiteral = "init"
+const awsInitCmdShortDesc = "Initialize a API project for a AWS API"
+const awsInitCmdLongDesc = `Downloading the OpenAPI specification of an API from the AWS API Gateway to initialize a WSO2 API project`
+const awsInitCmdExamples = utils.ProjectName + ` ` + awsCmdLiteral + ` ` + awsInitCmdLiteral  + ` -n PetStore -s Demo
+
+` + utils.ProjectName + ` ` + awsCmdLiteral + ` ` + awsInitCmdLiteral + ` --name PetStore --stage Demo
+
+` + utils.ProjectName + ` ` + awsCmdLiteral + ` ` + awsInitCmdLiteral + ` --name Shopping --stage Live
+
+NOTE: Both flags --name (-n) and --stage (-s) are mandatory as both values are needed to get the openAPI from AWS API Gateway.
+Make sure the API name and Stage Name are correct.
+Also make sure you have AWS CLI installed and configured before executing the aws init command.
+Vist https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html for more info`
+
+func getPath() {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return
+	}
+	dir = pwd
+}
+
+//aws init Cmd
+var InitCmd = &cobra.Command{
+	Use:   	awsInitCmdLiteral,
+	Short: 	awsInitCmdShortDesc,
+	Long:  	awsInitCmdLongDesc,
+	Example: 	awsInitCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		getPath()
+		initCmdOutputDir = dir + string(os.PathSeparator) + flagApiNameToGet
+
+		if stat, err := os.Stat(initCmdOutputDir); !os.IsNotExist(err) {
+			fmt.Printf("%s already exists\n", initCmdOutputDir)
+			if !stat.IsDir() {
+				fmt.Printf("%s is not a directory\n", initCmdOutputDir)
+				os.Exit(1)
+			}
+			if !awsInitCmdForced {
+				fmt.Println("Run with -f or --force to overwrite directory and create project")
+				os.Exit(1)
+			}
+			fmt.Println("Running command in forced mode")
+		}
+		execute()
+	},
+}
+
+type Apis struct {
+	Items []struct {
+		Id                    string `json:"id"`
+		Name                  string `json:"name"`
+	} `json:"items"`
+}
+
+func getOAS() error {
+	utils.Logln(utils.LogPrefixInfo + "Executing aws version command")
+	//check whether aws cli is installed
+	cmd_1, err := exec.Command(awsCmdLiteral, awsCLIVersionFlag).Output()
+	if err != nil {
+		fmt.Println("Error getting AWS CLI version. Make sure aws cli is installed and configured.")
```

Comment:
Shall we change this as below?
```suggestion
		fmt.Println("Error getting AWS CLI version. Make sure AWS CLI is installed and configured.")
```

Reply:
ack